### PR TITLE
ucr view permissions

### PR DESCRIPTION
**Pull request by calellowitz**
@sravfeyn @snopoke cc: @NoahCarnahan 
This models UCR view permissions off the ones currently used for other reports and fixes a broken link I found along the way

### CODE REVIEW

File: corehq/apps/userreports/reports/view.py [hunk 1]
```diff
@@ -274,7 +274,11 @@ def post(self, request, *args, **kwargs):
             raise Http403()
 
     def has_permissions(self, domain, user):
-        return True
+        if domain is None:
+            return False
+        if not user.is_active:
+            return False
+        return user.can_view_report(domain, 'corehq.reports.DynamicReport{}'.format(self.report_config_id))
```
Comment (calellowitz) [inline corehq/apps/userreports/reports/view.py]: ucrs when listed in the permissions object have the form `corehq.reports.DynamicReport{id}` which comes from the way the report classes are generated (and the fact that we store access to a report by class)

**Review by NoahCarnahan — APPROVED** [review]
👍

File: corehq/apps/locations/permissions.py [hunk 2]
```diff
@@ -95,7 +95,7 @@
     "This project has restricted data access rules. Please contact your "
     "project administrator to be assigned access to data in this project. "
     'More information is available <a href="{link}">here</a>.'
-).format(link="https://wiki.commcarehq.org/display/commcarepublic/Data+Access+Restrictions"))
+).format(link="https://wiki.commcarehq.org/display/commcarepublic/Data+Access+and+User+Editing+Restrictions"))
```
Comment (czue) [inline corehq/apps/locations/permissions.py]: already addresed in https://github.com/dimagi/commcare-hq/pull/14528/, though seems to merge cleanly.

File: corehq/apps/userreports/reports/view.py (hunk 1 above)
Comment (czue) [inline corehq/apps/userreports/reports/view.py]: not blocking but maybe a helper function with a good doc string might help with future maintenance of this?

**Review by czue — APPROVED** [review]
:boot:

File: corehq/apps/userreports/reports/view.py (hunk 1 above)
Comment (snopoke) [inline corehq/apps/userreports/reports/view.py]: @calellowitz would be good to follow up on this

**Pull request merged**